frontend/webservice/vrf.cgi

```diff
@@ -413,6 +413,12 @@ sub provision_vrf{
                 return;
             }
 
+            if(defined $interface->{provisionable_bandwidth} && !($ep->{bandwidth} + $interface->{utilized_bandwidth} <= $interface->{provisionable_bandwidth})){
```

Comment:
> The Interface object should have two methods, bandwidth and provisionable_bandwidth.

`$interface->{provisionable_bandwidth} ` should be `$interface->provisionable_bandwidth`

It should probably return `undef` if the interface is a non-cloud interface.

And FYI
```
sub method {
    return;
}
# returns `undef`

sub method {
    return undef;
}
# returns `[ undef ]`
```